internal/tray-agent/agent/client/agent-client.go

```diff
@@ -189,3 +209,26 @@ func AcquireKubeconfig() bool {
 	}
 	return true
 }
+
+//createKubeClient creates a new out-of-cluster client from a kubeconfig file.
+//If no value for kubeconfig is provided, it returns an error.
+//
+//The file path is retrieved from the env var specified by EnvLiqoKConfig.
+func createKubeClient() (kubernetes.Interface, error) {
+	if mockedController {
+		return fake.NewSimpleClientset(), nil
+	}
+	kubeconfig, ok := os.LookupEnv(EnvLiqoKConfig)
+	if !ok || kubeconfig == "" {
+		return nil, errors.New("no kubeconfig provided")
+	}
+	var (
+		cfg *rest.Config
+		err error
+	)
+	cfg, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
```

Comment:
why do you use the var block and don't use `cfg, err := ...` ?

Reply:
yes you're right, it has no particular meaning and maybe comes from an unfixed copy&paste or a previous version.